packages/renderer/index.d.ts

```diff
@@ -353,6 +223,169 @@ declare module '@react-pdf/renderer' {
 
     class Canvas extends React.Component<CanvasProps> {}
 
+    interface SVGProps extends NodeProps {
+      /**
+       * Enables debug mode on page bounding box.
+       * @see https://react-pdf.org/advanced#debugging
+       */
+      debug?: boolean;
+      width?: string | number;
+      height?: string | number;
+      viewBox?: string;
+      preserveAspectRatio?: string;
+    }
+
+    /**
+     * The <SVG /> element is a container that defines a new coordinate system and viewport. It is used as the outermost element of SVG documents.
+     */
+    class SVG extends React.Component<SVGProps> {}
```

Comment:
I guess it should be

```suggestion
    class Svg extends React.Component<SVGProps> {}
```